completion/available/github-cli.completion.bash

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+cite "about-completion"
+about-completion "GitHub CLI completion"
```

Comment:
Does this cover the same scope as the `gh` completion file? https://github.com/Bash-it/bash-it/blob/master/completion/available/gh.completion.bash

Reply:
I don't believe so as hub and github-cli are different packages. I imagine github-cli I might supersede hub at some point, but it's still fairly new.

Reply:
It looks like the `gh.completion.bash` file also sets up completion for the `gh` command, just like your new completion here...

https://github.com/Bash-it/bash-it/blob/dba8662a27cc04fca531db2b97f80ed6dc99164e/completion/available/gh.completion.bash#L363-L364

I'm wondering whether we should remove the `gh` completion file, since it seems to be generated, and probably is outdated anyway (6 years old already?)